src/components/plots/plot.ts

```diff
@@ -266,25 +287,26 @@ module Plottable {
      * Updates the extents associated with each attribute, then autodomains all scales the Plot uses.
      */
     protected _updateExtents() {
-      this._attrBindings.forEach((attr) => this._updateExtentsForAttr(attr));
+      this._attrBindings.forEach((attr) => this._updateExtentsForKey(attr, true));
+      this._propertyExtents.forEach((property) => this._updateExtentsForKey(property, false));
       this._scales().forEach((scale) => scale._autoDomainIfAutomaticMode());
     }
 
-    private _updateExtentsForAttr(attr: string) {
-      var binding = this._attrBindings.get(attr);
-      var accessor = binding.accessor;
-      var scale = binding.scale;
-      var coercer = (scale != null) ? scale._typeCoercer : (d: any) => d;
-      var extents = this._datasetKeysInOrder.map((key) => {
+    private _updateExtentsForKey(key: string, ifAttr: boolean) {
```

Comment:
I think this one should take the binding `Map` and the extent `Map` instead of a `boolean` -- makes it more readable when invoked.